extensions/che-theia-hosted-plugin-manager-extension/package.json

```diff
@@ -15,20 +15,36 @@
     "@eclipse-che/theia-remote-api": "^0.0.1"
   },
   "scripts": {
-    "prepare": "yarn clean && yarn build",
+    "prepare": "yarn clean && yarn build && yarn test",
```

Comment:
Why do we need to call `test` within the `prepare` script?

Reply:
We can create `"postbuild": "yarn test"` yarn script to run the tests. 

This was aligned to other approaches, we have:

- [recommendations-plugin/package.json#L38](https://github.com/eclipse/che-theia/blob/master/plugins/recommendations-plugin/package.json#L38) 
- [resource-monitor-plugin/package.json#L23](https://github.com/eclipse/che-theia/blob/master/plugins/resource-monitor-plugin/package.json#L23) 
- [workspace-plugin/package.json#L26](https://github.com/eclipse/che-theia/blob/master/plugins/workspace-plugin/package.json#L26) 
- [ports-plugin/package.json#L115](https://github.com/eclipse/che-theia/blob/master/plugins/ports-plugin/package.json#L115)